robottelo/ui/ldapauthsource.py

```diff
@@ -52,25 +60,14 @@ def create(self, name=None, server=None, ldaps=False, port=None,
 
     def search(self, name):
```

Comment:
The page doesn't contain search bar, all this method is doing is just navigating the page and selecting the right link. That's why no super call and base search (which is designed to work with search bar) can't be used.